components/createIdea/Form.jsx

```diff
@@ -0,0 +1,166 @@
+import React, { useState } from "react";
+import styles from "./From.module.scss";
+import { IoMdClose } from "react-icons/io";
+
+const Form = () => {
+  const [close, setClose] = useState(false);
+  const [tags, setTags] = useState([]);
+  const [errorFields , setErrorFields] = useState({})
+
+  const [formData, setFormData] = useState({
+    overview: "",
+    desc: "",
+    tags: "",
+    progress: "",
+  });
+
+  const changeHandler = (e) => {
+    setFormData({
+      ...formData,
+      [e.target.name]: e.target.value,
+    });
+  };
+
+  //form validation check
+  //validate function check if there exist empty field then update the errorField as true
+  const validate = (value) =>{
```

Comment:
what is the need of this validate function? if the use is just to set an input as required, it can be done using the `required` attribute.

docs: https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/required